ballista/rust/core/proto/ballista.proto

```diff
@@ -1179,9 +1179,21 @@ message ScalarValue{
         ScalarType null_list_value = 18;
 
         PrimitiveScalarType null_value = 19;
+        Decimal128 decimal128_value = 20;
+        int64 date_64_value = 21;
+        int64 time_second_value = 22;
+        int64 time_millisecond_value = 23;
+        int32 interval_yearmonth_value = 24;
+        int64 interval_daytime_value = 25;
     }
 }
 
+message Decimal128{
+  string value = 1;
```

Comment:
I'm confused about this point, we use the `String` type to hold the value of Decimal.

Reply:
> I'm confused about this point, we use the `String` type to hold the value of Decimal.

The value type is `i128` in datafusion Decimal128, But I don't know how to define i128 in proto. Or is it possible to use int64 in proto?

Reply:
Better use raw bytes.

Reply:
> Better use 16 raw bytes.

Yes, bytes is better. I will change the code

Reply:
done.